src/main/kotlin/commands/IssueCommand.kt

```diff
@@ -151,19 +152,29 @@ object IssueCommand : Command("issue") {
             return
         }
 
+        if (event.reaction.emoji.name == "❎") {
+            try {
+                message = message.error("Rejected")
+
+                form.first.delete()
+
+                delay(10000)
+
+                message.delete()
+            } catch (e: IllegalStateException) {
+                // ignored
+            }
+        }
+
         createGithubIssue(form.second, user, form.third, token)
 
         try {
             form.first.delete()
-        } catch (e: IllegalStateException) {
```

Comment:
these were intentionally individual, but now that commands have error handling you can just remove all of these